labels-owners.yaml

```diff
@@ -0,0 +1,157 @@
+
```

Comment:
Do we have labels for automl (katib), pytorch, xgboost and so on?

Reply:
added